lcls-twincat-motion/Library/DUTs/DUT_MotionStage.TcDUT

```diff
@@ -9,13 +9,22 @@ STRUCT
 	bLimBwd AT %I*: BOOL;
 	bHome AT %I*: BOOL;
 	bBrake AT %Q*: BOOL;
+	nEncCount	AT %I*	:	ULINT;
```

Comment:
The tabs in all of these lines make the document really difficult to review on github